test/IntegrationTests/test_listfile.py

```diff
@@ -0,0 +1,147 @@
+import unittest
+import requests
+import logging
+
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s - %(levelname)s - %(message)s'
+)
+logger=logging.getLogger(__name__)
+
+class TestListFileAPI(unittest.TestCase):
+    BASE_URL="http://localhost:8000"
+
```

Comment:
Add testcases like
1) Adding two files and verifying whether both the file names are listed
2) Adding the third file and verifying whether all three filenames are listed